doc/src/quickstart.md

```diff
@@ -0,0 +1,91 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
+
+We begin by creating a `Cargo.toml` file and load the necessary dependencies:
+
+```toml
+[package]
+name = "project"
+version = "0.0.1"
+authors = ["<authors>"]
+
+[[bin]]
+doc = false
+name = "project"
+
+[build-dependencies]
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+
+[dependencies]
+cfgrammar = { path="<path to cfgrammar>" }
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+```
+
+In the root of the project we create a file `build.rs`, which processes our
+lexer and grammar files. Here we can choose whether we want to generate a
```

Comment:
We should tell people where their grammar and lexer files should be stored.

Reply:
I was thinking about this `build.rs` file. I wonder if you could make this is a bit simpler for the user by providing a function that takes the action type, the lexer, and grammar files, and then does the `CTBuilder::new` and `LexerBuilder::new` thing for them. Or are there reasons why they need to see these steps?

Reply:
The point is that many users won't be able to use `lrlex` because regexs aren't powerful enough to lex their language (e.g. PHP). So `lrpar` is stand-alone because of that and thus `CTBuilder` shouldn't know anything about `lrlex` (though `lrlex` does know about `lrpar`).

Reply:
Ah, I see. That makes sense!